### PR TITLE
Added Language service

### LINKED ISSUE #607: Add Language Controls
DESCRIPTION.
Add a language service to translate content on tubular.

### PR DESCRIPTION
**Pull request by ManuSanRam**
This closes #607

WIP: DO NOT MERGE

### CODE REVIEW

**Comment by ManuSanRam** [comment]
WIP: DO NOT MERGE

File: src/DataGrid/NoDataRow.tsx [hunk 1]
```diff
@@ -14,7 +15,7 @@ export const NoDataRow: React.FunctionComponent<NoDataRowProps> = ({ columns, st
     <TableRow>
         <TableCell colSpan={columns.filter((col: ColumnModel) => col.visible).length}>
             <Typography style={styles.title} variant="body2" gutterBottom={true}>
-                <Warning /> No records found
+                <Warning /> {Lang.translate('NoRecords', 'USSPANISH')}
```
Comment (geoperez) [inline src/DataGrid/NoDataRow.tsx]: Us spanish?? Keep it simple. english.

File: src/Filtering/DialogInput.tsx [hunk 2]
```diff
@@ -73,9 +74,9 @@ export const DialogInput: React.FunctionComponent<DialogInputProps> = ({
     const disabled = isPrimary ? column.filter.operator === CompareOperators.None : false;
     const label = isPrimary
         ? column.filter.operator !== CompareOperators.Between
-            ? 'Value'
-            : 'First Value'
-        : 'Second Value';
+            ? `${Lang.translate('Label_Value', 'USSPANISH')}`
```
Comment (geoperez) [inline src/Filtering/DialogInput.tsx]: This implementation is random

File: src/utils/langService.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    public static translate(key: string, ...args: any[]): string {
+        let text = this.getTextByLanguage(key);
+        if (args.length > 0) {
+            text = this.formatTranslation(text, args);
+        }
+        return text;
+    }
+
+    private static getLangDataByCode(): any {
```
Comment (geoperez) [inline src/utils/langService.ts]: Why do you need this?

File: src/utils/langService.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import * as english from '../utils/languages/english.json';
+import * as spanish from '../utils/languages/spanish.json';
+
+const settings = { langName: 'english' };
```
Comment (geoperez) [inline src/utils/langService.ts]: Do you really need this? You may initialize Lang with English and that's it.

File: src/Pagination/Paginator.tsx [hunk 5]
```diff
@@ -21,10 +22,10 @@ const timeout = 400;
 
 const message = (totalRecordCount: number, filteredRecordCount: number) => ({ from, to, count }: any) =>
     totalRecordCount === filteredRecordCount
-        ? `${from}-${to} of ${count}`
+        ? Lang.translate('Pages', `${from}`, `${to}`, `${count}`)
         : filteredRecordCount === 0
-        ? '0 records found'
-        : `${from} to ${to} of ${count} from ${totalRecordCount} records`;
+        ? Lang.translate('NoRecords')
+        : Lang.translate('TotalRecords', `${from}`, `${to}`, `${count}`, `${totalRecordCount}`);
```
Comment (geoperez) [inline src/Pagination/Paginator.tsx]: Remove string interpolation from `...rest`

File: src/utils/langService.ts [hunk 6]
```diff
@@ -0,0 +1,39 @@
+import * as english from '../utils/languages/english.json';
+import * as spanish from '../utils/languages/spanish.json';
```
Comment (geoperez) [inline src/utils/langService.ts]: Don't initialize spanish

**Pull request merged**